src/timecorrelations.jl

```diff
@@ -62,6 +62,39 @@ function correlation(rho0::Operator, H::AbstractOperator, J,
 end
 
 
+"""
+    timecorrelations.correlation_dynamic(tspan, rho0, f, op1, op2; <keyword arguments>)
+
+
+Calculate two time correlation values ``⟨A(t)B(0)⟩`` for time-dependent Liouvillian
+
+The calculation is done by multiplying the initial density operator
+with ``B`` performing a time evolution according to a master equation
+and then calculating the expectation value ``\\mathrm{Tr} \\{A ρ\\}``
+
+
+# Arguments
+* `tspan`: Points of time at which the correlation should be calculated.
+* `rho0`: Initial density operator.
+* `f`: Function `f(t, rho) -> (H, J, Jdagger)` or `f(t, rho) -> (H, J, Jdagger, rates)`
+* `op1`: Operator at time `t`.
+* `op2`: Operator at time `t=0`.
+* `rates=ones(N)`: Vector or matrix specifying the coefficients (decay rates)
+        for the jump operators.
+* `kwargs...`: Further arguments are passed on to the ode solver.
+"""
+function correlation_dynamic(tspan, rho0::Operator, f, op1, op2;
+                             rates=nothing, kwargs...)
+
+    function fout(t, rho)
+        expect(op1, rho)
+    end
```

Comment:
I think it might be worth it to make this work also with a vector of `A`, because the heavy computation, doing the dynamics, is already done...?

Reply:
A dirty fix would be to change `expect(A, rho)` to `expect(rho,A)` (and also for `timecorrelations.correlation`), making use of  
`expect(op::AbstractOperator{B1,B2}, state::AbstractOperator{B2,B2}) where {B1,B2} = tr(op*state)
` and   
`expect(op::AbstractOperator, states::Vector) = [expect(op, state) for state=states]
` from `QuantumOpticsBase.jl
`.  
Or maybe it would be worth it to add a method like  
` expect(ops::Vector, states::AbstractOperator) = [expect(op, state) for op=ops] `
to allow `expect` to accept a vector of operators as first argument.